htcondor/htcondor_query.py

```diff
@@ -370,12 +367,14 @@ def split_collector_host(collector_host):
     collector.
     """
 
-    if isinstance(collector_host, str):
-        hosts = collector_host.replace(',', ' ').split()
-        primary = hosts[0]
-        secondary = hosts[1:]
-        secondary.sort()
-        return (primary, ','.join(secondary))
-    else:
-        RuntimeError(
-            'collector_host should be a comman or space separated string but found %s' % collector_host)
+    try:
+        if isinstance(collector_host, str):
+            hosts = collector_host.replace(',', ' ').split()
+            primary = hosts[0]
+            secondary = hosts[1:]
+            secondary.sort()
+            return (primary, ','.join(secondary))
+        else:
+            raise RuntimeError(f"collector_host should be a common or space separated string but found {collector_host}")
+    except Exception:
```

Comment:
since this looks like a defensive `except`, can you add `# pragma: no cover` to the `except Exception:` line?